src/outbackcdx/NanoHTTPD.java

```diff
@@ -1004,4 +1039,80 @@ public final InputStream getInputStream() {
             return bodyStream;
         }
     }
-}
\ No newline at end of file
+
+    // copied from guava
```

Comment:
We need to include guava's copyright notice and probably also the author tags from the javadoc (which would seem likely to count as an 'attribution notice'):

> You must retain, in the Source form of any Derivative Works that You distribute, all copyright, patent, trademark, and attribution notices from the Source form of the Work, excluding those notices that do not pertain to any part of the Derivative Works;

Reply:
How does this look?
https://github.com/nla/outbackcdx/pull/75/commits/9d29e742fe3e3e9d20a7170f8ecf8da73495f393

Reply:
Perfect